node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const writeFileButBetter = (data, encoder) => {
+  fs.writeFile(process.argv[3], data, encoder, err => {
+    if (err) throw err;
+  });
+};
+
+fs.readFile(process.argv[2], 'utf8', (err, data) => {
+  try {
+    writeFileButBetter('');
+  } catch (error) {
+    throw err;
```

Comment:
It's nice to see you playing around with try/catch. Note, though, that if an Error is not caught in a try/catch, it gets passed up to the next higher function in the call chain. So the try/catch you have here is not necessary.